src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/CalculationConfig.jsx

```diff
@@ -63,14 +71,6 @@ const CalculationConfig = (props) => {
     dispatch(loadEmbedding(experimentId, embeddingMethod));
   };
 
-  useEffect(() => {
-    if (!experimentId) {
-      return;
```

Comment:
why is this removed?

Reply:
It's not removed, it's just refactored into EmbeddingPreview, see https://github.com/biomage-ltd/ui/pull/97/files#diff-afd1e0b0de5b6da0610c8bd7fbb9865fec7ffc38aaf347805381ff303ea967bbR81-R86.